gibberish_server/models/room.js

```diff
@@ -38,7 +38,7 @@ class Room {
     toJSON() {
         let players = []
         for (let player of this.players) {
-            players.push(JSON.stringify(player));
+            players.push(player);
```

Comment:
the players field in room json is a list of json players, if you change it like this, then the for loop is not needed, I'm not sure what the json will look like on converting the list of players without making each a json itself